spec/changelog.yml

```diff
@@ -4,461 +4,464 @@
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/689
+    - description: Fix validation of empty arrays in YAML files
+      type: bugfix
+      link: https://github.com/elastic/package-spec/issues/683
+    - description: Dashboards with filters in all panels don't require a filter at the dashboard level
+      type: enhancement
+      link: https://github.com/elastic/package-spec/issues/687
+    - description: Allow volumes to be defined in custom-agent-deployer
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/689
 - version: 3.0.3
   changes:
-  - description: Do not require an object_type on objects with `enabled` set to `false`
-    type: bugfix
-    link: https://github.com/elastic/package-spec/issues/675
+    - description: Do not require an object_type on objects with `enabled` set to `false`
+      type: bugfix
+      link: https://github.com/elastic/package-spec/issues/675
 - version: 3.0.2
   changes:
-  - description: Add parquet files in terraform service deployer
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/662
-  - description: Require to define if a variable is a secret if it looks like a secret
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/665
-  - description: Allow _ in github owner field
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/668
+    - description: Add parquet files in terraform service deployer
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/662
+    - description: Require to define if a variable is a secret if it looks like a secret
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/665
+    - description: Allow _ in github owner field
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/668
 - version: 3.0.1
   changes:
-  - description: Using non-GA versions of the spec in GA packages produces a filterable validation error instead of a warning
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/627
-  - description: 'Allow to set index: false in dynamic templates defined in data stream manifests'
-    type: enhancement
-    link: https://github.com/elastic/package-spec/issues/650
-  - description: Validate that data streams with time series enabled have configured dimensions
-    type: enhancement
-    link: https://github.com/elastic/package-spec/issues/655
-  - description: Fix spec for rally benchmark scenario
-    type: bugfix
-    link: https://github.com/elastic/package-spec/pull/653
+    - description: Using non-GA versions of the spec in GA packages produces a filterable validation error instead of a warning
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/627
+    - description: "Allow to set index: false in dynamic templates defined in data stream manifests"
```

Comment:
There are many changes in formatting and quotes in this PR. I would prefer to avoid them if not needed.

Reply:
My linting picked it up changed it automatically, I noticed some other settings used double quotes. I will revert those formatting changes.